code/modules/cargo/randomstock.dm

```diff
@@ -0,0 +1,1880 @@
+			/obj/item/weapon/material/sword/trench,
+			/obj/item/weapon/material/sword/sabre
+			)
+
+			var/type = pick(swords)
+			new type(L)
+		if("ims")
+			new /obj/item/weapon/scalpel/manager(L)
+		if("hardsuit")
+			//A random RIG/hardsuit
+			//It will come with some screwy electronics and possibly need reprogramming
+			var/list/rigs = list(
+			"/obj/item/weapon/rig/unathi" = 2,
+			"/obj/item/weapon/rig/unathi/fancy" = 1,
+			"/obj/item/weapon/rig/combat" = 0.1,
+			"/obj/item/weapon/rig/ert" = 0.2,
+			"/obj/item/weapon/rig/ert/engineer" = 0.2,
+			"/obj/item/weapon/rig/ert/medical" = 0.3,
+			"/obj/item/weapon/rig/ert/security" = 0.15,
+			"/obj/item/weapon/rig/ert/assetprotection" = 0.1,
+			"/obj/item/weapon/rig/light" = 0.5,
+			"/obj/item/weapon/rig/light/hacker" = 0.8,
+			"/obj/item/weapon/rig/light/stealth" = 1.5,
+			"/obj/item/weapon/rig/merc" = 1,
+			"/obj/item/weapon/rig/industrial" = 3,
+			"/obj/item/weapon/rig/eva" = 3,
+			"/obj/item/weapon/rig/ce" = 2,
+			"/obj/item/weapon/rig/hazmat" = 4,
+			"/obj/item/weapon/rig/medical" = 4,
+			"/obj/item/weapon/rig/hazard" = 4,
+			)
+
+			var/type = pickweight(rigs)
+			var/obj/item/weapon/rig/module = new type(L)
+
+			//screw it up a bit
+			var/cnd = rand(1,100)
+			module.lose_modules(cnd)
+			module.misconfigure(cnd)
+			module.sabotage_cell()
+			module.sabotage_tank()
+
+
+		//Several random non-weapon exosuit attachments
+		if ("exogear")
+			var/list/equips = list(
+			"/obj/item/mecha_parts/mecha_equipment/tool/hydraulic_clamp," = 1,
+			"/obj/item/mecha_parts/mecha_equipment/tool/drill," = 1,
+			"/obj/item/mecha_parts/mecha_equipment/tool/drill/diamonddrill," = 0.7,
+			"/obj/item/mecha_parts/mecha_equipment/tool/extinguisher" = 1,
+			"/obj/item/mecha_parts/mecha_equipment/tool/rcd" = 0.08,
+			"/obj/item/mecha_parts/mecha_equipment/teleporter" = 0.3,
+			"/obj/item/mecha_parts/mecha_equipment/wormhole_generator" = 0.5,
+			"/obj/item/mecha_parts/mecha_equipment/gravcatapult" = 0.8,
+			"/obj/item/mecha_parts/mecha_equipment/anticcw_armor_booster" = 1,
+			"/obj/item/mecha_parts/mecha_equipment/antiproj_armor_booster" = 0.9,
+			"/obj/item/mecha_parts/mecha_equipment/repair_droid" = 0.7,
+			"/obj/item/mecha_parts/mecha_equipment/tesla_energy_relay" = 0.4,
+			"/obj/item/mecha_parts/mecha_equipment/generator" = 1.5,
+			"/obj/item/mecha_parts/mecha_equipment/generator/nuclear" = 0.8,
+			"/obj/item/mecha_parts/mecha_equipment/tool/safety_clamp" = 0.2,
+			"/obj/item/mecha_parts/mecha_equipment/tool/passenger" = 1,
+			"/obj/item/mecha_parts/mecha_equipment/tool/sleeper" = 0.9,
+			"/obj/item/mecha_parts/mecha_equipment/tool/cable_layer" = 1.2,
+			"/obj/item/mecha_parts/mecha_equipment/tool/syringe_gun" = 1
+			)
+
+			var/number = rand(2,5)
+			while (number > 0)
+				var/type = pickweight(equips)
+				new type(L)
+
+		if ("teleporter")
+			new /obj/item/weapon/hand_tele(L)
+
+		if ("voice")
+			new /obj/item/clothing/mask/gas/voice(L)
```

Comment:
There's nothing inherently wrong with a device that alters your voice. We have them nowadays, some people use them for voice RP in online games and chatrooms

It just happens to be a thing thats useful for espionage too